pkg/keymanagementprovider/azurekeyvault/provider.go

```diff
@@ -177,17 +201,30 @@ func (s *akvKMProvider) GetKeys(ctx context.Context) (map[keymanagementprovider.
 			return nil, nil, fmt.Errorf("failed to get key objectName:%s, objectVersion:%s, error: %w", keyVaultKey.Name, keyVaultKey.Version, err)
 		}
 
+		if keyVaultKey.Version == "" {
+			keyVaultKey.Version = getObjectVersion(*keyBundle.Key.Kid)
+		}
+
```

Comment:
nit: we can add `isEnabled := *keyBundle.Attributes.Enabled` here to save one line.